src/content/about/index.mdx

```diff
@@ -15,8 +15,8 @@ One Acquisition Solution for Integrated Services Plus or OASIS+ is a suite of go
 
 ## A streamlined program with expanded offerings
 
-This suite of contracts combines the scope of legacy OASIS; Building, Maintenance &
-Operations; and Human Capital & Training Solutions under one program, along with new areas of service that were previously not in scope. It expands the range of service offerings for buyers and gives sellers a greater variety of areas they can compete for.
+This suite of contracts combines the scope of legacy OASIS, Building, Maintenance, and
```

Comment:
I just want to double check that this is what you intended. The sentence:
"This suite of contracts combines the scope of legacy OASIS, Building, Maintenance, and Operations and Human Capital and Training Solutions, under one program, along with new areas of service that were previously not in scope" seems like we have four things under the program:

- legacy OASIS
- Building
- Maintenance
- Operations and Human Capital and Training Solutions

but I read the original as three things:

- legacy OASIS
- Building Maintenance and Operations
- Human Capital and Training Solutions

Reply:
This was the content Rosamund suggested but I agree with you. Let's make it crystal clear and use the following:

This suite of contracts combines the scope of legacy OASIS – Building, Maintenance, & Operations and Human Capital & Training Solutions – under one program, along with new areas of service that were previously not in scope